tests/mvp-demo/mvp-demo.go

```diff
@@ -247,16 +245,14 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 		})
 
 		It("Snapshot is created", func() {
-			Eventually(func() bool {
+			Eventually(func() (bool, error) {
 				snapshot, err = f.AsKubeAdmin.IntegrationController.GetSnapshot("", "", componentName, userNamespace)
-				if err != nil || len(snapshot.Name) == 0 {
+				if err != nil {
 					GinkgoWriter.Printf("cannot get the Snapshot: %v\n", err)
-					return false
+					return false, err
 				}
-
-				GinkgoWriter.Printf("snapshot %s is found\n", snapshot.Name)
-				return true
-			}, snapshotTimeout, snapshotPollingInterval).Should(BeTrue(), "timed out when trying to check if the Snapshot exists")
+				return true, nil
+			}, snapshotTimeout, snapshotPollingInterval).Should(BeTrue())
```

Comment:
@gabemontero @Dannyb48 @dheerajodha 

after this code change this is how the test output looks like when the `Eventually` loop in this spec fails
```
• [FAILED] [240.553 seconds]
[mvp-demo-suite MVP Demo tests] MVP Demo Chapter 1 - basic build & deploy, failed release [It] Snapshot is created [mvp-demo, mvp-demo-chapter-1]
/Users/psturc/work/redhat-appstudio/e2e-tests/tests/mvp-demo/mvp-demo.go:247

  [FAILED] Timed out after 240.001s.
  The function passed to Eventually returned the following error:
      <*errors.errorString | 0xc000118980>:
      no snapshot found for component 'componentName', pipelineRun '' in 'mvp-dev-hbgn-tenant' namespace
      {
          s: "no snapshot found for component 'componentName', pipelineRun '' in 'mvp-dev-hbgn-tenant' namespace",
      }
  In [It] at: /Users/psturc/work/redhat-appstudio/e2e-tests/tests/mvp-demo/mvp-demo.go:255 @ 05/19/23 09:20:14.523
```
wdyt?

Reply:
@gabemontero I changed that on purpose only to simulate the error so I can share how the error output looks like (apologies, forgot to mention that 🙂  )

Reply:
I like this approach, thanks! :D